tools/functional-tester/etcd-tester/etcd_runner_stresser.go

```diff
@@ -0,0 +1,64 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os/exec"
+
+	"syscall"
+)
+
+type runnerStresser struct {
+	cmd    *exec.Cmd
+	cmdStr string
+	args   []string
+
+	errorc chan error
+}
+
+func (rs *runnerStresser) setupOnce() (err error) {
+	if rs.cmdStr == "" {
+		return fmt.Errorf("expect cmdStr to be set")
+	}
+	if rs.cmd != nil {
+		return nil
+	}
+
+	rs.cmd = exec.Command(rs.cmdStr, rs.args...)
+	stderr, err := rs.cmd.StderrPipe()
+
+	go func() {
+		out, err := ioutil.ReadAll(stderr)
```

Comment:
if I understand runner code correctly, runner will exit on panic, log.panic, log.fataf, and `ExitWithError` which all of those will write to stderr and exit the program. it is possible that runner runs forever if it never encounters any problems. However, when the tester do clean up, tester will kill the runner which will close stderr.